.stylelintrc.json

```diff
@@ -12,7 +12,7 @@
     },
     "no-eol-whitespace": true,
     "function-parentheses-space-inside": "never-single-line",
-    "max-nesting-depth": 5,
+    "max-nesting-depth": 4,
```

Comment:
What do these do/mean?

Reply:
https://stylelint.io/user-guide/rules/list/max-nesting-depth/
https://stylelint.io/user-guide/rules/list/selector-max-compound-selectors/

And we have this item to work on these, cause they are connected with our general goal of reducing specificity level too
https://github.com/iTwin/iTwinUI/issues/333

I think we want to end up these rules having value of 3. They will help also for new components/features, so we develop those straight away in a good level.